### PR TITLE
LPS-175576 changed filter popover to dropdown

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/_liferay.scss [hunk 1]
```diff
@@ -93,6 +71,8 @@ body {
 	max-height: 800px;
 	width: 300px;
 
+	overflow-y: hidden;
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/styles/_liferay.scss:74]: Change alphabetical order and remove unnecessary line

```suggestion
	overflow-y: hidden;
```

**Review by samuelleolopes — CHANGES_REQUESTED** [review]

**Review by Elvinho — APPROVED** [review]
LGTM 👍🏾

**Review by rafaelfs98 — APPROVED** [review]
LGTM :fire:

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

**Comment by welitonpe** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by welitonpe** [comment]
ci:stop

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**